demos/tests/args.py

```diff
@@ -27,9 +27,22 @@ def __init__(self, rel_path):
     def resolve(self, context):
         return str(context.test_data_dir / self.rel_path)
 
+class TMP:
```

Comment:
I think you've committed some WIP stuff by accident.

Reply:
Yes, there is `tmp` (747dfb4) commit. I will drop it from history.

Reply:
dropped